libs/ui-lib/lib/ocm/components/clusterWizard/OperatorsStep.tsx

```diff
@@ -22,7 +26,18 @@ export const OperatorsStep = (props: ClusterOperatorProps) => {
       <StackItem>
         <MceCheckbox />
       </StackItem>
-      <StackItem>{isSNO ? <LvmCheckbox {...props} /> : <OdfCheckbox />}</StackItem>
+      {isVersionEqualsOrMajorThan4_15 ? (
```

Comment:
I just thought of this.

For SNO with version >= 4.15, both operators can be also chosen?

Reply:
Question asked to @gamli75 in ticket

Reply:
LVMS should be enabled in SNO same as today also in 4.15 (response from BE)